### ISSUE TITLE
TypeError: store() got an unexpected keyword argument 'safe'

### ISSUE THREAD

**Issue by wujun4code** [yacir/markdown-fenced-code-tabs#16]
### 1. clone this repo

```sh
$ git clone https://github.com/yacir/markdown-fenced-code-tabs.git
```

### 2. mkdocs serve

```sh
$ cd docs/
$ mkdocs serve
```

### 3. error

```sh
INFO    -  Building documentation...
WARNING -  Config value: 'pages'. Warning: The 'pages' configuration option has been deprecated and will be removed in a future release of MkDocs. Use 'nav' instead.
INFO    -  Cleaning site directory
ERROR   -  Error reading page 'index.md': store() got an unexpected keyword argument 'safe'
Traceback (most recent call last):
  File "/usr/local/bin/mkdocs", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/__main__.py", line 134, in serve_command
    livereload=livereload
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/serve.py", line 119, in serve
    config = builder()
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/serve.py", line 114, in builder
    build(config, live_server=live_server, dirty=dirty)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/build.py", line 274, in build
    _populate_page(file.page, config, files, dirty)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/commands/build.py", line 177, in _populate_page
    page.render(config, files)
  File "/usr/local/lib/python2.7/site-packages/mkdocs/structure/pages.py", line 184, in render
    self.content = md.convert(self.markdown)
  File "/usr/local/lib/python2.7/site-packages/markdown/core.py", line 265, in convert
    self.lines = prep.run(self.lines)
  File "/usr/local/lib/python2.7/site-packages/markdown_fenced_code_tabs/__init__.py", line 204, in run
    return self._render_code_tabs(parsed_lines).split('\n')
  File "/usr/local/lib/python2.7/site-packages/markdown_fenced_code_tabs/__init__.py", line 152, in _render_code_tabs
    transformed_lines += '\n' + self.markdown.htmlStash.store(group_html, safe=True) + '\n\n'
TypeError: store() got an unexpected keyword argument 'safe'
```

### Environment

- OS: Mac OS Mojave

```sh
$  mkdocs --version
mkdocs, version 1.0.4 from /usr/local/lib/python2.7/site-packages/mkdocs (Python 2.7)
```

**Comment by katharine-adaptive** [comment]
Recommended fix: 
[#safe_mode-and-html_replacement_text-keywords-deprecated](https://python-markdown.github.io/change_log/release-3.0/#safe_mode-and-html_replacement_text-keywords-deprecated)

**Comment by katharine-adaptive** [comment]
<a html="https://python-markdown.github.io/change_log/release-3.0/#safe_mode-and-html_replacement_text-keywords-deprecated">Recommended fix</a>

**Comment by csm10495** [comment]
Temp fix for those who can't wait:
```
python -m pip install -U markdown==2.6.11 --force-reinstall
```
Not pretty but just downgrades the underlying markdown library

**Comment by yacir** [comment]
Sorry for the delay! Fixed in the `1.0.4`.
